### PR TITLE
fix(core): Prevent `__default__` jobs in scaling mode

### PR DESCRIPTION
**Pull request by ivov**
## Summary

See details in story.

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/PAY-2414

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/cli/src/workflow-runner.ts [hunk 1]
```diff
@@ -66,10 +66,15 @@ export class WorkflowRunner {
 		//
 		// FIXME: This is a quick fix. The proper fix would be to not remove
 		// the execution from the active executions while it's still running.
-		if (error instanceof ExecutionNotFoundError || error instanceof ExecutionCancelledError) {
+		if (
+			error instanceof ExecutionNotFoundError ||
+			error instanceof ExecutionCancelledError ||
+			error.message.includes('cancelled')
```
Comment (ivov) [inline packages/cli/src/workflow-runner.ts:72]: Not ideal, but see: https://linear.app/n8n/issue/PAY-2414/cancellation-in-scaling-mode-leads-to-job-being-re-enqueued#comment-f257b343

**Review by netroy — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.74.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.74.1)

**Comment by janober** [comment]
Got released with [`n8n@1.74.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.74.1)

**Comment by janober** [comment]
Got released with [`n8n@1.74.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.74.0)

**Pull request merged**